src/noos_tf/cli.py

```diff
@@ -30,8 +30,8 @@ def run(ctx, message="", workspace="", organisation=None, token=None):
 
 
 ns = Collection()
-ns.add_task(update)
-ns.add_task(run)
+ns.add_task(Task(update))
+ns.add_task(Task(run))
 
 
 main = Program(namespace=ns, version=__version__)
```

Comment:
💡 

To avoid defining the project version in its root package (`noos_tf.__version__`) and in its `pyproject.toml`, maybe the use of `importlib`?

```
from importlib import metada


version = metadata.version("noos-tf")
```